client/src/theme.js

```diff
@@ -731,27 +731,27 @@ const theme = {
   paragraph: {
     large: {
       height: '19px',
-      maxWidth: '235px',
+      maxWidth: '100%',
```

Comment:
does it has the same effect if we remove this value?

Reply:
I wanted to keep it there for now as a reminder it exists

Reply:
But i think it does because it would be undefined and the default behavior for a `p` would be to expand to fill it's container

Reply:
well, expand as it's content dictates until wrapping after filling the container

Reply:
set it to undefined to remove the extra rule applied